data/human/jobs.txt

```diff
@@ -2099,6 +2147,54 @@ mission "Wealthy tourists back"
 		payment 10000 200
 		dialog phrase "generic passenger dropoff payment"
 
+mission "Wealthy tourist group back [0]"
+	name "Wealthy tourists home to <planet>"
+	job
+	repeat
+	description "A group of wealthy tourists, consisting of <bunks> members, are headed home to <destination>, and wish to travel in a luxurious ship for the journey. They are willing to pay you <payment>."
+	passengers 15 5 .09
+	to offer
+		random < 75
+		random < ( "passenger space" - 35 ) * 2
```

Comment:
Are these double random conditions meant as a cap on the offer rate of those missions that increase in offer rate with passenger space? If so, why are some of the missions capable of reaching a 100% offer rate and others aren't?

Reply:
Yeah, the double randoms are meant to cap the global high-passenger jobs. The more regional/time-limited jobs don't have this cap because they're already restricted enough.